msteams-platform/index.yml

```
@@ -14,100 +15,123 @@ metadata:
-    - title:  Microsoft Teams and Microsoft Graph API
+      url: /microsoftteams/platform/concepts/design/understand-use-cases
+    - title: Design your app
       itemType: overview
+      url: /microsoftteams/platform/concepts/design/design-teams-app-overview
+    - title: Build your app 
+      itemType: how-to-guide
+      url: /microsoftteams/platform/overview#start-building
+    - title: Enhance your app with Microsoft Graph
+      itemType: how-to-guide
       url:  /graph/teams-concept-overview
-    - title:  Low-code apps, bots, flows, and reports using Microsoft Power Platform
-      itemType: overview
-      url:  /microsoftteams/platform/samples/teams-low-code-solutions/
+    - title: Publish your app to the Teams store
+      itemType: deploy
+      url: /microsoftteams/platform/concepts/deploy-and-publish/appsource/publish
+    - title: Tutorials and code samples
+      itemType: sample
+      url:  /microsoftteams/platform/tutorials/code-samples
+    - title: What's new?
+      itemType: whats-new
+      url: /microsoftteams/platform/whats-new
+
    
 # conceptualContent section
 conceptualContent:
 # itemType: architecture | concept | deploy | download | get-started | how-to-guide | learn | overview | quickstart | reference | sample | tutorial | video | whats-new
    title: Guides and Resources
    items:
-   # Card
-    - title: Explore
+
+  # Card
+    - title: Audience
       links:
-        - url: /microsoftteams/platform/concepts/building-an-app
-          itemType: concept
-          text: Build a Microsoft Teams app
-        - url: /microsoftteams/platform/concepts/extensibility-points
+        - url: /microsoftteams/microsoft-teams
           itemType: concept
-          text: Capability points in Microsoft Teams
-        - url: /microsoftteams/platform/concepts/design/understand-use-cases
+          text: Developers
+        - url: /microsoftteams/microsoft-teams
           itemType: concept
-          text: Understand Microsoft Teams use cases
-        - url:  /microsoftteams/platform/concepts/design/map-use-cases
+          text: IT professionals
+        - url: https://support.microsoft.com/en-us/office/apps-and-services-cc1fba57-9900-4634-8306-2360a40c665b?ui=en-US&rs=en-US&ad=US#PickTab=Basics
           itemType: concept
+          text: End users
+
+   # Card
+    - title: Explore
+      links:
+        - url: /microsoftteams/platform/concepts/capabilities-overview
+          itemType: overview
+          text: Teams app capabilities
+        - url: /microsoftteams/platform/concepts/design/map-use-cases
+          itemType: overview
           text: Map use cases to Microsoft Teams capabilities
-        - url:  /microsoftteams/platform/whats-new 
-          itemType: whats-new
-          text: What's new
+        - url: /microsoftteams/platform/tutorials/get-started-nodejs-app-studio
+          itemType: how-to-guide
+          text: Build a Teams app using JavaScript/Node.js and App Studio
```

```suggestion
          text: Build a Teams app using Node.js
```
More concise and Aligns more closely with docs title.